ui/partials/perf/comparechooserctrl.html

```diff
@@ -13,7 +13,7 @@
             <label><input type="checkbox" ng-model="revisionComparison">Compare with a specific revision</label>
           </div>
           <div ng-show="!revisionComparison">
-            <p class="help-block">By default, Perherder will compare against performance data gathered over the last 2 days</p>
+            <p class="help-block">By default, Perherder will compare against performance data gathered over the last 2 days from when new revision was pushed</p>
```

Comment:
Nit: there's a legacy typo 'Perherder' (although with the loading cat gif maybe purrherder would work hah)

Reply:
lol, I will fix this as a quick followup